instance/github.py

```diff
@@ -158,13 +159,8 @@ def __init__(self, number, source_fork_name, target_fork_name, branch_name, titl
         self.title = title
         self.username = username
         self.body = body
-
-    @property
-    def extra_settings(self):
-        """
-        Extra settings contained in the PR body
-        """
-        return get_settings_from_pr_body(self.body)
+        self.extra_settings = get_settings_from_pr_body(body)
+        self.extra_settings_dict = yaml.load(self.extra_settings) or {}
```

Comment:
@antoviaque My viewpoint here was that `PR` instances

(a) are ephemeral.  They are created, used and dropped, but not kept around for a longer time or ever changed.

(b) are duplicating data anyway, since the golden version of the data is on Github, so even if we do keep them around, we have to keep them in sync with the version on Github.  It would be easy to make sure all fields are updated when resyncing with Github.

The motivation of this change is that it becomes quite computationally expensive if we need to parse the whole PR body for the settings, extract them and run the YAML parser everytime we want to access a setting.  I'm happy to turn them into properties again, but I'd like to remind you that the last bug I fixed was caused by too many things being properties and recomputed everytime when actually they shouldn't be, so you have to look at each individual case to figure out the right approach.